File: src/main/java/com/dy/dyweb/bean/Comment.java
package com.dy.dyweb.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Comment {

    // 评论Id
    private Long commentId;

    // 在该评论下接着评论
    private Long parentCommentId;

    // 用户发布评论Id
    private Long userId;

    // @的用户集合
    private List<Long> referredUserId;

    // 回复的用户Id
    private Long replyUserId;

    // 评论内容
    private String content;

    // 评论时间
    private LocalDateTime createTime;

}

File: data/src/main/java/com/aking/data/model/Comment.kt
package com.aking.data.model

data class Comment(
    // 评论Id
    val commentId: Long,
    // 作品Id
    val worksId: Long,
    // 在该评论下接着评论
    val parentCommentId: Long,
    // 用户发布评论Id
    val userId: Long,
    // 回复的用户Id
    val replyUserId: Long,
    // @的用户集合
    val referredId: List<Long>,
    // 评论内容
    val content: String? = null,
    // 评论时间
    val createTime: String? = null,
    val page: Int ?= 0,
    val rows: Int ?= 0
)
